community/php7/APKBUILD

```diff
@@ -535,6 +539,14 @@ phar() {
 	fi
 }
 
+default_exts() {
+	pkgdesc="PHP$_suffix metapackage for all Core Extensions and Bundled Extensions"
+	depends=$(printf '%s\n' $_extensions \
+		| sed -En "s/(\w+)\*/$pkgname-\1/p")
+
+	mkdir -p "$subpkgdir"
+}
```

Comment:
Sorry for confusion, I’ve merged all commits from this PR _except_ the last one adding php7-default-exts. I’ll merge this separately after we resolve how to name it.